diagnostic.js

```diff
@@ -41,7 +84,9 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
-
+db.houses.remove(
```

Comment:
no, this removes every house that has that motto, we just want to remove the motto from that one house.

```js
db.houses.update({ name: "House Arryn"}, { $unset: { motto: 'anything'} });
```